### PR TITLE
Feature/reordering algo properties

### PR DESCRIPTION
**Pull request by salmma**
reordering properties in overview for
- algorithms
- implementations
- cloud services

solves #152

### CODE REVIEW

**Review by mbeisel — COMMENTED** [review]
Algorithm overview: lgtm
Implementations: Name, contributors, version and license have a rendering buttons that look out of place.
Cloud platforms: lgtm

**Review by mbeisel — APPROVED** [review]
Looks good!

File: src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html [hunk 1]
```diff
@@ -20,13 +21,80 @@
         ></app-text-input>
       </mat-card>
     </div>
+  </div>
+
+  <div class="row" *ngIf="algorithm.computationModel === 'QUANTUM' && frontendAlgorithm.computationModel === 'QUANTUM'">
+    <!-- TODO: finish Sketch -->
```
Comment (lharzenetter) [inline src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html:27]: What's with this TODO?

**Review by lharzenetter — COMMENTED** [review]
Besides TODO, LGTM

File: src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html (hunk 1 above)
Comment (salmma) [reply src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html:27]: (it is an old comment) I think the idea was to add a sketch component (e.g. for showing circuits, as in the platform) supporting the upload of images.

Comment (lharzenetter) [reply src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html:27]: If this is not realized yet we should add it as an issue to track this todo.

**Review by lharzenetter — APPROVED** [review]

File: src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html (hunk 1 above)
Comment (salmma) [reply src/app/components/algorithms/algorithm-properties/algorithm-properties.component.html:27]: Here it is https://github.com/UST-QuAntiL/qc-atlas-ui/issues/164

**Pull request merged**